### ISSUE TITLE
[POSIX] can't find ::unlink symbol

### ISSUE THREAD

**Issue by southorange0929** [Tencent/MMKV#1233]
## Note:
An crash issue will be **ignored & closed** in a week **without logs**.

### The language of MMKV
Try to use `core` to build with `HarmonyOS`, but get some error

<img width="1814" alt="image" src="https://github.com/Tencent/MMKV/assets/32590310/286cbc71-626b-443e-b526-b699e990a477">

### The version of MMKV
master branch

### The platform of MMKV

harmonyos

### The installation of MMKV

cmake

### What's the issue?

Try to build source code with harmonyos, but failed. Look like that needs to add `unistd` header file.

Should we add this header into `MMKV_IO.cpp` file?

cmake
```
# the minimum version of CMake.
cmake_minimum_required(VERSION 3.4.1)
project(MyApplication2)

set(NATIVERENDER_ROOT_PATH ${CMAKE_CURRENT_SOURCE_DIR})

if(DEFINED PACKAGE_FIND_FILE)
    include(${PACKAGE_FIND_FILE})
endif()

include_directories(${NATIVERENDER_ROOT_PATH}
                    ${NATIVERENDER_ROOT_PATH}/include)

add_subdirectory(MMKV/Core)

add_library(entry SHARED hello.cpp)
target_link_libraries(entry PUBLIC libace_napi.z.so core)
```

**Comment by lingol** [comment]
Yep, sure.

**Comment by southorange0929** [comment]
Can you publish a release version? Thanks

**Comment by lingol** [comment]
A posix port doesn't really rely on official release because it is mainly used by sources. So a new release won't be needed because of this bug.

**Comment by southorange0929** [comment]
Is this commit will merged to master branch?

**Comment by lingol** [comment]
It will, eventually.

**Comment by lingol** [comment]
It's released with v1.3.4.
